homework-1/index.html

```diff
@@ -0,0 +1,681 @@
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Scout</title>
+  <link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css" />
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <div id="banner-submenu" class="fixed-top">
+    <nav class="submenu">
+      <ul class="submenu__list list-top" id="list-top">
+        <li class="first-item"><a class="tips-active" href="#tips">Tips</a></li>
+        <li class="submenu__item-middle"><a class="submenu__link" href="#glossary">Glossary</a></li>
+        <li class="last-item"><a href="#fqas">FAQs</a></li>
+      </ul>
+    </nav>
+  </div>
+  <header class="header" id="header">
+    <div class="logo">
+      <a class="logo__link" href="#">
+        skout
+      </a>
+    </div>
+    <div class="header__menu menu">
+      <div class="menu__icon">
+        <span></span>
+      </div>
+      <nav class="menu__body">
+        <ul class="menu__list">
+          <li class="link-hover-center"><a class="menu__link first-link" href="#">Why Skout </a></li>
+          <li class="link-hover-center"><a class="menu__link" href="#">Products</a></li>
+          <li class="link-hover-center"><a class="menu__link" href="#">Partners</a></li>
+          <li class="link-hover-center"><a class="menu__link" href="#">Company</a></li>
+          <li class="link-hover-center active"><a class="menu__link" href="#">Tips & Tools</a></li>
+          <li class="link-hover-center"><a class="menu__link" href="#">Blog</a></li>
+          <div class="header__button-link">
+            <button type="button"></button>
+            <a class="get__link" href="#">Get Skout</a>
+            <a class="log__link" href="#">Log In</a>
+          </div>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <div class="header__banner" id="banner">
+
+    <h1 class="header__banner-title">Tips & Tools</h1>
+    <div class="header__banner-submenu banner-top">
+      <nav class="submenu">
+        <ul class="submenu__list">
+          <li class="first-item"><a class="tips-active link-active" href="#tips">Tips</a></li>
+          <li class="submenu__item-middle"><a class="submenu__link" href="#glossary">Glossary</a></li>
+          <li class="last-item"><a href="#fqas">FAQs</a></li>
+        </ul>
+      </nav>
+    </div>
+  </div>
+  <!----------tips-start----------->
+  <section class="tips section" id="tips">
+    <div class="container">
+
+      <div class="tips__wrrapper">
+        <div class="tips__img border-img">
+          <img class="mobile_img" src="images/logo-tips_mobile.png" alt="tips">
+          <img class="dekstop_img" src="images/logo-tips.png" alt="tips">
+        </div>
+        <ul class="tips-list">
+          <li class="border-item">
+            <button type="button" class="tips__link tips__link-first">
+              <span class="border-radius"></span>
```

Comment:
It is better to add styles to button and you can avoid redundant tags.

Reply:
next time I will try.